checkstyle.xml

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE module PUBLIC
+        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+        "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="AvoidStarImport"/>
+    </module>
+<!--    <module name="LineLength">-->
```

Comment:
This needs to be discussed and either removed or addressed step by step, as it is currently not compliant.